pkg/asset/installconfig/nutanix/nutanix.go

```diff
@@ -265,6 +268,9 @@ func getSubnet(ctx context.Context, client *nutanixclientv3.Client, peUUID strin
 
 func getVIPs() (string, string, error) {
 	var apiVIP, ingressVIP string
+	var installConfig types.InstallConfig
+	installConfig.Platform = types.Platform{Nutanix: &nutanix.Platform{}}
+	defaults.SetInstallConfigDefaults(&installConfig)
```

Comment:
nit: It might be simpler/less confusing to export `defaultMachineCIDR` from package `types/defaults`; because I wonder if this implies "we're creating the install config here", which we're not.

Reply:
yeah, I was torn on that tbh.  My concern was that based on the platform, that defaultMachineCIDR might not be the actual defaultMachineCIDR.  While this doesn't impact any current onprem platforms, I was concerned this might cause some confusion down the line.  another option here might be to pull the logic out of SetInstallConfigDefaults that determines the final defaultMachineCIDR of the platform.

Reply:
thinking about this a little more, I think i'll go ahead and revert to your suggestion here.